FirebaseFirestore.podspec

```diff
@@ -98,7 +99,7 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   s.dependency 'abseil/time', abseil_version
   s.dependency 'abseil/types', abseil_version
 
-  s.dependency 'gRPC-C++', '~> 1.28.0'
+  s.dependency 'GRPCSwiftShim', '~> 0.0.1'
```

Comment:
Why is the Shim a separate library versus part of Firestore itself?

Reply:
Because it is just a quick hack prototype.